src/sunpos.jl

```diff
@@ -74,6 +74,11 @@ function sunpos(jd::AbstractVector{J}; radians::Bool=false) where {J<:Real}
     return ra, dec, longmed, oblt
 end
 
+sunpos(dt::DateTime; radians::Bool=false) =
+    sunpos(datetime2julian(dt); radians)
```

Comment:
The implicit keyword argument isn't compatible with Julia v1.0, but I'd be ok with bumping the required julia version at this point.

Reply:
```suggestion
    sunpos(datetime2julian(dt); radians=radians)
```
Easy enough to fix